polyfills/Promise/config.json

```diff
@@ -14,7 +14,7 @@
 		"bb": "* - 10",
 		"chrome": "* - 31",
 		"firefox": "6 - 28",
-		"ie": "8 - *",
+		"ie": "< 12",
```

Comment:
At the lower end, you want to set the version constraint to the first version of the browser that is capable of applying the polyfill. This is currently set to 8 for Promise in IE, which seems to be right, since our CI shows it doesn't pass tests in IE7.  At the higher end, you want to set it to the last version of the browser that doesn't have the feature you're polyfilling, and that's where I think you want to make the change, right?

So you probably want `8 - 12` (which will include everything <13.0.0)